bin/adhoc-scripts/updatePileupObjects.py

```diff
@@ -19,20 +22,28 @@
 import ssl
 from http.client import HTTPSConnection
 
+# WMCore services
+from Utils.Timers import gmtimeSeconds
+
 
 class OptionParser():
     """Class to parse the command line arguments"""
 
     def __init__(self):
         "User based option parser"
+        defaultDN = '/DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=msunmer/CN=852819/CN=Robot: WmCore Service Account'
```

Comment:
I actually think we should not persist any DNs in github. So I would suggest to fix the relevant records in MongoDB (testbed and production), then we update this pull request before merging it.

Reply:
removed